components/kyma-environment-broker/internal/storage/storage_test.go

```diff
@@ -1134,104 +1132,44 @@ func fixInstance(testData instanceData) *internal.Instance {
+
+	return &instance
 }
 
 func fixProvisionOperation(testData string) internal.ProvisioningOperation {
-	return internal.ProvisioningOperation{
-		Operation: fixSucceededOperation(testData),
-	}
-}
-func fixDeprovisionOperation(testData string) internal.DeprovisioningOperation {
-	return internal.DeprovisioningOperation{
-		Operation: fixSucceededOperation(testData),
-	}
-}
+	operationId := fmt.Sprintf("%s-%d", testData, rand.Int())
+	return internal.FixProvisioningOperation(operationId, testData)
 
-func fixSucceededOperation(testData string) internal.Operation {
-	return internal.Operation{
-		ID:                     fmt.Sprintf("%s-%d", testData, rand.Int()),
-		CreatedAt:              fixTime().Add(24 * time.Hour),
-		UpdatedAt:              fixTime().Add(48 * time.Hour),
-		InstanceID:             testData,
-		ProvisionerOperationID: testData,
-		State:                  domain.Succeeded,
-		Description:            testData,
-		ProvisioningParameters: internal.ProvisioningParameters{},
-	}
 }
-
-func fixTime() time.Time {
-	return time.Date(2020, 04, 21, 0, 0, 23, 42, time.UTC)
+func fixDeprovisionOperation(testData string) internal.DeprovisioningOperation {
+	operationId := fmt.Sprintf("%s-%d", testData, rand.Int())
+	return internal.FixDeprovisioningOperation(operationId, testData)
 }
 
 func fixRuntimeOperation(operationId string) orchestration.RuntimeOperation {
-	return orchestration.RuntimeOperation{
-		ID: operationId,
-		Runtime: orchestration.Runtime{
-			ShootName:              "shoot-stage",
-			MaintenanceWindowBegin: time.Now().Truncate(time.Millisecond).Add(time.Hour),
-			MaintenanceWindowEnd:   time.Now().Truncate(time.Millisecond).Add(time.Minute).Add(time.Hour),
-			RuntimeID:              "runtime-id",
-			GlobalAccountID:        "global-account-if",
-			SubAccountID:           "subaccount-id",
-		},
-		DryRun: false,
-	}
+	runtime := internal.FixRuntime("runtime-id")
+	runtimeOperation := internal.FixRuntimeOperation(operationId)
+	runtimeOperation.Runtime = runtime
+
+	return runtimeOperation
 }
 
 func fixProvisioningParameters() internal.ProvisioningParameters {
-	active := true
-	return internal.ProvisioningParameters{
-		PlanID:    broker.TrialPlanID,
-		ServiceID: broker.KymaServiceID,
-		ErsContext: internal.ERSContext{
-			TenantID:        "test",
-			SubAccountID:    "test",
-			GlobalAccountID: "test",
-			ServiceManager: &internal.ServiceManagerEntryDTO{
-				Credentials: internal.ServiceManagerCredentials{
-					BasicAuth: internal.ServiceManagerBasicAuth{
-						Username: "username",
-						Password: "password",
-					}},
-			},
-			Active: &active,
-		},
-		Parameters: internal.ProvisioningParametersDTO{
-			Name:        "test",
-			KymaVersion: "0.0.0",
-		},
-		PlatformRegion: "region",
-	}
+	pp := internal.FixProvisioningParameters("test")
+	pp.PlanID = broker.TrialPlanID
+	pp.ServiceID = broker.KymaServiceID
+	return pp
 }
 
 func fixInstanceDetails() internal.InstanceDetails {
```

Comment:
`fixInstanceDetails` can also be moved to fixture.go

Reply:
Will do.